seqr/management/commands/check_bam_cram_paths.py

```diff
@@ -49,3 +53,12 @@ def handle(self, *args, **options):
             logger.info('{} files not found:'.format(sum(missing_counter.values())))
             for project_name, c in sorted(missing_counter.items(), key=lambda t: -t[1]):
                 logger.info('   {} in {}'.format(c, project_name))
+
+            # post to slack
+            slack_message = 'Found {} broken bam/cram path(s)\n'.format(sum(missing_counter.values()))
+            for project_name, missing_paths_list in project_name_to_missing_paths.items():
+                slack_message += "\nIn project {}:\n".format(project_name)
+                slack_message += "\n".join([
+                    "  {}   {}".format(individual_id, path) for individual_id, path in missing_paths_list
+                ])
+            safe_post_to_slack(SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL, slack_message)
```

Comment:
can you update the unit test to test the slack notifications

Reply:
by mocking `safe_post_to_slack`?

Reply:
yep!